tests/stub/session_run_parameters/test_session_run_parameters.py

```diff
@@ -145,7 +141,7 @@ def test_impersonation(self):
                           })
 
     @driver_feature(types.Feature.IMPERSONATION,
-                    types.Feature.BOLT_4_4)
+                    types.Feature.BOLT_4_3)
```

Comment:
Just checking. Does this mean that both 4.4 and 4.3 are required?

Reply:
No. I've changed it from only requiring 4.4 to only requiring 4.3.

Reply:
Never mind what I said... It means that, yes. I fixed it.